spec/integration/data_stream/manifest.spec.yml

```diff
@@ -307,6 +332,29 @@ spec:
         dynamic_namespace:
           description: When set to true, agents running this integration are granted data stream privileges for all namespaces of its type
           type: boolean
+    routing_rules:  # technical preview
```

Comment:
Tried to install a package with this new routing_rules key in 8.7.1 and it is installed successfully. No error and no reference to those processors neither.

Should we add some validation rule for this ?

Reply:
Probably, it could be kept as it is without validation rule